src/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright Olivier Lamy
+ * Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Oh yes this one is a rename of an existing one.
But I guess for the new class I created Http2ConnectorFactory I can use the license I prefer as a creator?